Zagrios/bs-manager
# Feature/readme update

## Scope



[closes TICKET-45](https://github.com/Zagrios/bs-manager/issues/45)

## Implementation



- Recreating a readme for the 1.0 release

## Screenshots

| before | after |
| ------ | ----- |
| ![image](https://user-images.githubusercontent.com/40648115/213424968-ca5b0be7-9dbd-41ec-b6c7-f9f5f22e45f9.png)| ![image](https://user-images.githubusercontent.com/40648115/213425156-3556440a-82d2-48de-b2c2-65089e75b63d.png)|

## How to Test



- [x] Review (by another) : 
> - [x] Check check if all the links have been set and working 
> - [x] Check if there are any missing sections
> - [x] Check Orthography
> - [x] Check Grammar

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |
